kalite/distributed/tests/browser_tests/quiz.py

```diff
@@ -8,8 +8,11 @@
 
 from kalite.testing.base import KALiteBrowserTestCase
 from kalite.testing.mixins import BrowserActionMixins, FacilityMixins
+from selenium.webdriver.common.keys import Keys
 
-PLAYLIST_ID = "g3_p1"
+import urllib
```

Comment:
Follow https://github.com/learningequality/ka-lite/wiki/Coding-guidelines-and-conventions#imports